MIGRATION.md

```diff
@@ -531,6 +532,18 @@ Starting in 8.0, Storybook requires Svelte 4 and up.
 
 The `--use-npm` is now removed. Use `--package-manager=npm` instead. [More info here](#cli-option---use-npm-deprecated).
 
+#### LinkTo direct import from addon-links
+
+The `LinkTo` (React component) direct import from `@storybook/addon-links` is now removed. You have to import it from `@storybook/addon-links/react` instead.
+
+```ts
+// before
+import LinkTo from '@storybook/addon-links';
```

Comment:
Is the default Import intended here? Or should it be a named import?

Reply:
There was never a named import for this component